src/main/scala/com/gradle/SbtCommonCustomUserDataPlugin.scala

```diff
@@ -1,37 +1,44 @@
 package com.gradle
 
 import com.gradle.develocity.agent.sbt.DevelocityPlugin
-import com.gradle.develocity.agent.sbt.DevelocityPlugin.autoImport.DevelocityConfiguration
-import com.gradle.internal.{CustomBuildScanEnhancements, ServerConfigTemp, Overrides}
+import com.gradle.develocity.agent.sbt.DevelocityPlugin.autoImport._
```

Comment:
is the `_` like a `*` import for Java? let's avoid those an import everything explicitly.